src/formatDate.js

```diff
@@ -7,8 +7,62 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparatorFormat = fromFormat[3];
+  const toSeparatorFormat = toFormat[3];
+  const dateObject = {};
+
+  const arrayFromDate = date.split(fromSeparatorFormat);
```

Comment:
Consider destructuring the `arrayFromDate` directly within the `split` operation to improve readability and reduce the need for indexing later on. For example: `const [year, month, day] = date.split(fromSeparatorFormat);`